### PR TITLE
Added 4 Band Resistance Calculator

### LINKED ISSUE #66: 4 Band Resistance Calculator
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

Overview
The Calculator will calculate the theoritical value of Resistance of a resistor using the 4 colour bands present on it.

Interface of Calculator
The UI of Calculator will have:
a) 4 drop down sections to select the colour of the bands.
b) Submit button

Functioning of Calulator
The calculator then will return the output resistance + tolerance value of the selected colour bands of a resistor with appropriate units inside a popup box.

Please assign this issue to me to work under the program of SocialWinterofCode'24
Thank you!!

### Screenshots

_No response_

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

### PR DESCRIPTION
**Pull request by amansgith**
# Fixes Issue🛠️
#66 4 Band Resistance Calculator

Closes #66 

# Description👨‍💻 

The 4 Band Resistance Calculator is a simple and interactive that calculates the theoritical Resistance Value of a Resistor.
1. It has 4 drop downs to selection band colours.
2. Sumbit button to calculate the resistance value.
3. Output is shown in a popup box.
4. The Calculator is Responsive.

## Some Additional changes
1. Improved readability of index.html by adding comments
2. Fixed the bug of extra logo appearing in the footer section as shown in the below image.
bug

# Type of change📄

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist✅ 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added demonstration in the form of GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷
## Demo 1
demo
## Demo 2
demo1
## Demo 3
demo2
## Demo 4
demo3
## Demo 5
demo4
## Demo 6
demo5

### CODE REVIEW

**Comment by Rakesh9100** [comment]
Pls resolve the conflicts.
Write our folder name using `hyphen` instead of `underscore`.
Do remove the template code as it is not needed since we just have to add a section which everyone is able to @amansgith

**Comment by amansgith** [comment]
Sure i have fixed the issues.. just left with changing the folder name

**Comment by amansgith** [comment]
@Rakesh9100 Please check now.. i have resolved the conflicts and did as you suggested
Thank you for you patience.

File: index.html [hunk 1]
```diff
@@ -249,15 +255,30 @@ <h3>Calculates the Pressure conversion between two specified units.</h3>
                 </a>
             </div>
         </div>
+        <div class="box">
+            <div class="content">
+                <h2>4 Band Resistance Calculator</h2>
+                <h3>Calculates the theoritical resistance value of a Resistor having 4 Colour Bands.</h3>
+                <a href="./Calculators/4_Band_Resistance_Calculator/index.html" target="_blank">
```
Comment (Rakesh9100) [inline index.html:262]: Pls change the `underscore` to `hyphen` here @amansgith

Comment (amansgith) [reply index.html]: Done

**Comment by Rakesh9100** [comment]
Good job, Appreciated! Merged!! Congrats on getting your PR merged to this repo under SWOC S4 🥳🎉 @amansgith

**Pull request merged**